worldedit-core/src/main/java/com/fastasyncworldedit/core/FaweAPI.java

```diff
@@ -60,17 +59,6 @@
 
     private static final Logger LOGGER = LogManagerCompat.getLogger();
 
-    /**
-     * Offers a lot of options for building an EditSession.
-     *
-     * @return A new EditSessionBuilder
-     * @see EditSessionBuilder
-     */
-    @Deprecated
-    public static EditSessionBuilder getEditSessionBuilder(World world) {
-        return new EditSessionBuilder(world);
-    }
-
```

Comment:
Removing this method is an API breakage. Even if it's marked as deprecated, people still may use it, as it used to be the way-to-go when working with FAWE. I'd rather mark it for removal, fix the javadoc linking at `@see` and delegate it to the new method.